### DISCUSSION TITLE
storage:link issue

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#991] [Q&A (Help)]
Have you ever had an error like this:

The strange thing is that only "composer install" makes this error, but if I enter ssh with the same system user I execute the "php artisan storage:link" command without errors!
What can it be ?

**Comment by karandatwani92** [comment]
Hey @blondie63 

Please attach the error screenshot. it seems you missed uploading it.

**Reply by blondie63** [reply]
<img width="929" alt="Screenshot 2024-05-19 alle 18 52 05" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/f330169c-baea-4409-bcfa-4a3bc3df200e">

i deploy using an automatic system that create a new release from git and some scripts to enable it
because there's this error i do this test:
inside the new folder i run "composer install" that fail with error (screenshot above coming from script log, but it's the same)
on same folder i run php artisan storage:link and there's no errors and symlink is created 

very strange issue

**Reply by karandatwani92** [reply]
Hey @blondie63 

This question is related to your server configuration & Laravel. I can't help much here.
I found some help on this here:
- https://stackoverflow.com/questions/47409417/storage-in-laravel-says-symlink-no-such-file/47902653#47902653
- https://stackoverflow.com/questions/70381201/call-to-undefined-function-illuminate-filesystem-symlink

I hope it helps you solve it.

**Comment by blondie63** [comment]
Problem is other and as workaround i deleted cmd from composer.json and put in in my deploy script, after composer install
